packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -185,7 +185,7 @@ class Matrix4Tween extends Tween<Matrix4> {
     end!.decompose(endTranslation, endRotation, endScale);
     final Vector3 lerpTranslation =
         beginTranslation * (1.0 - t) + endTranslation * t;
-    // TODO(alangardner): Implement slerp for constant rotation
+    // TODO(alangardner): Implement lerp for constant rotation
```

Comment:
@Lioness100 @goderbauer Not sure, but this could be a mistake, since `slerp` is a common shorthand for "spherical linear interpolation", which is used when linearly interpolating quaternions. See https://en.wikipedia.org/wiki/Slerp